_stbt/core.py

```diff
@@ -1770,11 +1772,15 @@ def _find_matches(image, template, match_parameters, imglog):
         raise ValueError("Template image must be 8-bits per channel")
 
     # pylint:disable=undefined-loop-variable
-    for first_pass_matched, region, first_pass_certainty in \
-            _find_candidate_matches(image, template, match_parameters, imglog):
+    for i, first_pass_matched, region, first_pass_certainty in \
+            _find_candidate_matches(image, template, match_parameters,
+                                    imglog):
         confirmed = (
             first_pass_matched and
-            _confirm_match(image, region, template, match_parameters, imglog))
+            _confirm_match(image, region, template, match_parameters,
+                           imwrite=lambda name, img: imglog.imwrite(
+                               "match%d-%s" % (i, name), img)))  # pylint:disable=cell-var-from-loop
```

Comment:
This would be a bit nicer with the logging proposal I made in #361.  Not important, just a data point.